openmc/source.py

```diff
@@ -0,0 +1,119 @@
+from numbers import Real
+import sys
+from xml.etree import ElementTree as ET
+
+from openmc.stats.univariate import Univariate
+from openmc.stats.multivariate import UnitSphere, Spatial
+import openmc.checkvalue as cv
+
+if sys.version_info[0] >= 3:
+    basestring = str
+
+
+class Source(object):
+    """Distribution of phase space coordinates for source sites.
+
+    Parameters
+    ----------
+    space : openmc.stats.Spatial, optional
+        Spatial distribution of source sites
+    angle : openmc.stats.UnitSphere, optional
+        Angular distribution of source sites
+    energy : openmc.stats.Univariate, optional
+        Energy distribution of source sites
+    filename : str, optional
+        Source file from which sites should be sampled
+    strength : Real
+        Strength of the source
+
+    Attributes
+    ----------
+    space : openmc.stats.Spatial or None
+        Spatial distribution of source sites
+    angle : openmc.stats.UnitSphere or None
+        Angular distribution of source sites
+    energy : openmc.stats.Univariate or None
+        Energy distribution of source sites
+    file : str or None
+        Source file from which sites should be sampled
+    strength : Real
```

Comment:
Is it true that the `strength` is really an implied probability for a multinomial distribution over multiple `Source` objects?